scripts/kong_backend/swap_sol_to_usdc.sh

```
@@ -0,0 +1,189 @@
+NETWORK_FLAG=""
+if [ "${NETWORK}" != "local" ]; then
+    NETWORK_FLAG="--network ${NETWORK}"
+fi
+IDENTITY="--identity kong_user1"
+KONG_BACKEND=$(dfx canister id ${NETWORK_FLAG} kong_backend)
+
+# Token configurations
+PAY_TOKEN="SOL"
+PAY_AMOUNT=10_000_000  # 0.01 SOL (9 decimals)
+PAY_AMOUNT=${PAY_AMOUNT//_/}  # remove underscore
+SOL_CHAIN="SOL"
```

Shellcheck (0.10.0)

[warning] SOL_CHAIN appears unused. Verify use (or export if used externally).

(SC2034)